### PR TITLE
Pass in s3path to Notifier; add tests

### PR DESCRIPTION
**Pull request by YinfangZ**

### CODE REVIEW

File: lib/aurora_bootstrapper/exporter.rb [hunk 1]
```diff
@@ -5,13 +5,18 @@ class Exporter
...
-      @notifier           = Notifier.new
+      begin
+        @match              = "#{prefix}.*"
+        @export_bucket      = export_bucket
+        @blacklisted_tables = blacklisted_tables.split(",")
+        @whitelisted_tables = whitelisted_tables.split(",")
+        @blacklisted_fields = blacklisted_fields.split(",")
+        @client             = client
+        @notifier           = Notifier.new( s3_path: export_bucket )
+      rescue => e
```
Comment (gaorlov) [inline lib/aurora_bootstrapper/exporter.rb:16]: ruby lets you skip the `begin` for block-level checks, so you can do 

```ruby
def initialize
  # init stuff
rescue => e
  # log error
  raise e
 end
 ```

File: lib/aurora_bootstrapper/exporter.rb [hunk 2]
```diff
@@ -11,7 +11,10 @@ def initialize( client:, prefix: "", export_bucket:, blacklisted_tables: "", whi
       @whitelisted_tables = whitelisted_tables.split(",")
       @blacklisted_fields = blacklisted_fields.split(",")
       @client             = client
-      @notifier           = Notifier.new
+      @notifier           = Notifier.new( s3_path: export_bucket )
+    rescue => e
+      AuroraBootstrapper.logger.error message: "Error in initializing exporter.", error: e
+      raise
```
Comment (gaorlov) [inline lib/aurora_bootstrapper/exporter.rb:17]: let's re-raise `e` here

Comment (YinfangZ) [reply lib/aurora_bootstrapper/exporter.rb:17]: Sure thing.

**Review by gaorlov — APPROVED** [review]
thanks for cleaning this up!

**Pull request merged**